src/main/java/io/microconfig/osdf/paths/OSDFPaths.java

```diff
@@ -44,6 +44,10 @@ public Path componentsPath() {
         return of(root() + "/buildConfigs");
     }
 
+    public Path jmeterPath() {
+        return of(componentsPath() + "/openshift-jmeter");
```

Comment:
better make special folder for this type of system components 
for example: .osdf/system/...